lib/node/rpc.js

```diff
@@ -1104,6 +1105,27 @@ class RPC extends RPCBase {
     }
   }
 
+  async compactTree(args, help) {
+    if (help || args.length !== 0)
+      throw new RPCError(errs.MISC_ERROR, 'compacttree');
```

Comment:
I'm not getting the meaning of this bit.

Reply:
Take a look at the other rpc functions. They all could use work but what this tells the user is the usage of the command very simply (accepts no arguments)

Reply:
Ah ok, just seemed weird for it to be an exception with no explanation info in the message but knowing the convention it makes sense.